### ISSUE TITLE
Fake strikes damage gloves

### ISSUE THREAD

**Issue by l29ah** [CleverRaven/Cataclysm-DDA#52803]
### Describe the bug

> You fake a strike at the shorthair cat
> Your |\ pair of light gloves is damaged by the force of the blow!

### Steps To Reproduce

Load the game and try hitting the cat.

### Expected behavior

No glove damage from misses.

### Versions and configuration

- OS: Linux
    - OS Version: <unknown>
- Game Version: 0.F-3826-g610e7d5502-dirty [64-bit]
- Graphics Version: Curses
- Game Language: English [en]
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    No Fungal Growth [no_fungal_growth],
    Bionic Professions [package_bionic_professions]
]

### Additional context

[2021.11.10.zip](https://github.com/CleverRaven/Cataclysm-DDA/files/7532838/2021.11.10.zip)

**Comment by LeahLuong** [comment]
You can't fake a strike @ a shorthair cat w/out SOME convincing motion behind it!

That said, I dented my brass knuckles the other day by missing a Z. I think the game has a flat chance for wielded item damage dependent on the item & regardless of outcome. It would be nice if it was more nuanced.

**Comment by jumjummju** [comment]
In general I think missed strikes causing damage to your weapon makes sense, since a "missed strike" can include over-swinging and hitting the floor or a nearby wall or the like. It is a bit silly with unarmed strikes though unless you're just going full Donkey Kong and smashing the ground. Maybe you could reason it as punching them in the tooth or some other boney protrusion that doesn't do any damage but snags the glove.

**Comment by Photoloss** [comment]
> In general I think missed strikes causing damage to your weapon makes sense, since a "missed strike" can include over-swinging and hitting the floor or a nearby wall or the like. It is a bit silly with unarmed strikes though unless you're just going full Donkey Kong and smashing the ground. Maybe you could reason it as punching them in the tooth or some other boney protrusion that doesn't do any damage but snags the glove.

No, hitting bone would count as being deflected by natural armour, or possibly blocked by technique if NPCs can do that. A miss is a miss, and means the weapon did not make contact with the target at all. Also relevant for zapping effects.

Unless you suffer additional effects such as falling over the risk of overswinging only applies to weapons which are actually long enough to reach the ground. This is the case for most "real" weapons but precisely in the early game where damage matters most (due to lack of easy replacements) many improvised weapons are shorter. Especially knives, rocks and most tools.

**Comment by Amoebka** [comment]
As a somewhat related issue, clothing can also be damaged by attacks that failed to hit it. Chainmail armor is too durable to be damaged by basic zombies, but when the attack bypasses the 95% coverage, it has a chance to damage the armor regardless, which is really weird.

**Comment by Paragonid** [comment]
Damaging the weapon (or gloves) during fake strikes or misses is critical failure rate rare for any decently skill fighter. 
You can check any online fighting videos for the proof, sport competitions included. 

The problem in the game representing the issue is even worse when trying to hit small agile creature like bat. Currently player will completely destroy own tactical gloves trying to hit the bat
